packages/medusa-payment-stripe/src/services/stripe-bancontact.js

```diff
@@ -1,15 +1,13 @@
-import _ from "lodash"
 import Stripe from "stripe"
-import { PaymentService } from "medusa-interfaces"
+import { AbstractoPaymentService } from "@medusajs/medusa"
+import { PaymentSessionData } from "@medusajs/medusa/src/interfaces"
+import { Cart } from "@medusajs/medusa/src/models"
 
-class BancontactProviderService extends PaymentService {
+class BancontactProviderService extends AbstractoPaymentService {
```

Comment:
issue: Typo in import. Is your IDE not complaining about it? 🤔 
```suggestion
class BancontactProviderService extends AbstractPaymentService {
```

Reply:
No 😂 it is a js file maybe thats why it does not say anything, the pipeline did not get any trouble to build as well